### PR TITLE
Support djangocms-picture 2.0.0 and higher in create_picture_plugin

### PR DESCRIPTION
**Pull request by leture**
djangocms-picture 2.0.0 switched to a filer reference to store the image so the helper must create a filer image instance

Refs #642

### CODE REVIEW

**Comment by leture** [comment]
@fsbraun Because of the lack of an existing test to just adjust I wrote a test for the helper `extract_images` which subsequently also tests `create_picture_plugin`. This seemed to be more worthwhile than just a direct call to `create_picture_plugin`. So it is tested that the image extraction logic is actually executed on a text plugin save.

**Comment by leture** [comment]
@fsbraun I am finished here.

**Review by fsbraun — APPROVED** [review]
Very nice! Looks good to me. I have just a minor recommendation. Thank you so much, @leture!

File: tests/test_plugin.py [hunk 1]
```diff
@@ -40,6 +40,13 @@
     HAS_DJANGOCMS_TRANSLATIONS = False
 
 
+try:
+    import djangocms_picture  # noqa
+    HAS_DJANGOCMS_PICTURE = True
+except ImportError:
+    HAS_DJANGOCMS_PICTURE = False
+
+
```
[the remark this answers is not in the document]
Comment (leture) [reply tests/test_plugin.py]: Adjusted it.

**Pull request merged**